src/lua_console.cpp

```diff
@@ -0,0 +1,93 @@
+#include "lua_console.h"
+
+#include "catalua.h"
+#include "catacurse.h"
```

Comment:
Don't include this, include "cursesdef.h". "catacurses.h" contains the declaration for the curses emulation used by SDL / native windows builds. Their content conflicts heavily with the content of "ncurses.h", which is included in builds with curses. "cursesdef.h" conditionally includes whichever is required.

Btw. it's nice to see somebody adding new source files to those CodeBlocks files. Please also add them to "src/CMakeLists.txt".